### PR TITLE
correcting typos and word spelling

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

**Comment by SamMarkowitz** [comment]
Thanks for picking up the typos. There are quite a few of them. My bad. 

However, the "subflow", "walkthrough", and "subfolder" instances are written that way on purpose. They seem to be accepted versions of their hyphenated counterparts and they make the text flow a little better.

**Comment by lirantal** [comment]
Cool, I'll change those back and commit.

**Comment by lirantal** [comment]
@SamMarkowitz  I changed them back

File: app/markdowns/slang.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ CloudSlang is a [YAML](http://www.yaml.org) (version 1.2) based language for des
 See the [New Hire tutorial](http://cloudslang-tutorials.readthedocs.org) for a comprehensive walkthrough of the CloudSlang language's features.
 
 ##YAML Overview
-Before writing CloudSlang code it helps to have a good working knowledge of YAML. The following is a brief overview of some of the YAML specification. See the full [YAML specification](http://www.yaml.org/spec/1.2/spec.html) for more information.
+Before writing CloudSlang code it helps to have a good working knowledge of YAML. The following is a brief overview of some of the YAML specification. See the full [YAML specificationn](http://www.yaml.org/spec/1.2/spec.html) for more information.
```
Comment (SamMarkowitz) [inline app/markdowns/slang.md]: Is this one a mistake? Two n's?

**Comment by lirantal** [comment]
@SamMarkowitz good catch, I updated it.

**Pull request merged**